src/MediaBundle/Controller/MediaController.php

```diff
@@ -349,6 +349,15 @@ public function createAction(Request $request)
 
             $em->flush();
 
+            if ($this->get('security.context')->isGranted('ROLE_ADMIN')) {
```

Comment:
Should be `ROLE_USER`, and exclude `ROLE_ADMIN` so the admin don't send himself a mail.